.buildkite/commands/prototype-build-jetpack.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -eu
 
-# FIXIT-13.1: Prototype Builds want the latest version of Sentry CLI
-brew update
+# Sentry CLI needs to be up-to-date
 brew upgrade sentry-cli
```

Comment:
While self-reviewing the code, I realized this comment is not clear enough.

First, we need the CLI installed, and our lanes will fail if there isn't because the call `sentry_check_cli_installed`, e.g.: https://github.com/wordpress-mobile/WordPress-iOS/blob/1be5d8cf6be34a2245400a2164375d123fb161b6/fastlane/lanes/build.rb#L120

But, the `sentry_check_cli_installed` action doesn't necessarily fails if the CLI is not at the _latest_ version, only if it isn't at a version compatible with the Fastlane plugin where the action comes from. See the [action description](https://github.com/getsentry/sentry-fastlane-plugin/blob/f93ea306bc35f8f4161b5bb1aa644666bc62c398/lib/fastlane/plugin/sentry/actions/sentry_check_cli_installed.rb#L17-L22):

> This action checks that the senty-cli is installed and meets the mimum verson requirements.

Reply:
Thinking about it, given the minimum CLI version is determined by [what Sentry bundles in its Fastlane plugin](https://github.com/getsentry/sentry-fastlane-plugin/blob/f93ea306bc35f8f4161b5bb1aa644666bc62c398/lib/fastlane/plugin/sentry/helper/sentry_helper.rb) the only instance in which this might fail is if we upgrade the plugin to a version that requires a new CLI. But, since we are in charge of when to upgrade the plugin, we can prevent or correct this failure and wait to upgrade till a new VM image is available. Alternatively, we can add the `brew upgrade sentry-cli` command if we absolutely need the new Sentry plugin version and make a note to remove once a new VM image is out.

Reply:
Interestingly, the latest CLI version is 2.13.0 but the `brew upgrade sentry-cli` command in CI [doesn't find it](https://buildkite.com/automattic/wordpress-ios/builds/13050#0186da74-1684-4523-a125-84b060d8b288/248-533) 🤔 

![image](https://user-images.githubusercontent.com/1218433/224867386-5525dc3d-553b-491b-8bd6-3b3e52fb43e1.png)

Reply:
I may open a followup PR to either update the wording of that comment, or remove the whole Sentry CLI management based on the rationale from [the previous comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/20316/files#r1134775489).

Reply:
> but the brew upgrade sentry-cli command in CI [doesn't find it](https://buildkite.com/automattic/wordpress-ios/builds/13050#0186da74-1684-4523-a125-84b060d8b288/248-533)

Maybe we need to call `brew update` before `brew upgrade sentry-cli`, so it fetches the latest formulae first? 🤔

Reply:
@AliSoftware Good point. The logs I linked came from a release build, so it already run with `brew update` commented...

Indeed, in builds that run `brew update` the Sentry CLI updates as expected

![image](https://user-images.githubusercontent.com/1218433/225170286-7db6e1b0-c0e3-4605-adc9-ef729647b869.png)


Notice that my first comment said the latest Sentry CLI version was 2.13.0, but this screenshot shows 2.14.4. What I should have written originally is "the latest version bundled in the latest Fastlane plugin is 2.13.0".